--- f2fs-tools-1.9.0/lib/Makefile.am.orig	2017-10-15 16:19:55.961880191 +0200
+++ f2fs-tools-1.9.0/lib/Makefile.am	2017-10-15 16:20:35.875230392 +0200
@@ -6,3 +6,4 @@ libf2fs_la_SOURCES = libf2fs.c libf2fs_i
 libf2fs_la_CFLAGS = -Wall
 libf2fs_la_CPPFLAGS = -I$(top_srcdir)/include
 libf2fs_la_LDFLAGS = -version-info $(LIBF2FS_CURRENT):$(LIBF2FS_REVISION):$(LIBF2FS_AGE)
+libf2fs_la_LIBADD = -lblkid
